app/build.gradle

```diff
@@ -1,19 +1,19 @@
 apply plugin: 'com.android.application'
-apply plugin: 'com.neenbedankt.android-apt'
 
 def commitCount = Integer.parseInt('git rev-list --all --count'.execute([], project.rootDir).text.trim())
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.2"
+    compileSdkVersion 26
+    buildToolsVersion "26.0.2"
     defaultConfig {
         applicationId "com.lucasurbas.masterdetail"
         minSdkVersion 21
-        targetSdkVersion 25
+        targetSdkVersion 26
         versionCode commitCount
         versionName "1.1.0"
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
         resConfigs "en"
+        vectorDrawables.useSupportLibrary = true
```

Comment:
Min sdk is 21 so I believe this is not needed

Reply:
Yeah, I forgot to remove it, because I played a little bit with SDK < 21